File: Daily_Programs/Python practice/palindrome.py
print("Enter number to check if it is a palindrome number:")
n=int(input())
num=n
rnum=int(0)
while num!=0:
    d=int(num%10)
    rnum=(rnum*10)+d
    num=int(num/10)

if(n==rnum):
    print(n ,"Is palindrome number")
else:
    print(n ,"Is not palindrome number")        

File: Daily_Programs/C practice/palindrome.c
#include <stdio.h>
int main()
{
    int n;
    printf("Enter the number: ");
    scanf("%d", &n) ;
    int num = n;
    int rev = 0;
    while (num > 0) {
        int d = num % 10;
        rev = rev * 10 + d;
        num /= 10;
    }
    if (rev == n)
        printf("Palindrome Number\n");
    else
        printf("Not a Palindrome Number\n");
    return 0;
}